### PR TITLE
Add examples of tests to README

### LINKED ISSUE #12: Update readme to include examples of tests
Include examples of tests using the packages and building out WC related e2e tests.

### PR DESCRIPTION
**Pull request by zhongruige**
This PR adds in examples of using the testing packages.

Closes https://github.com/woocommerce/woocommerce-e2e-boilerplate/issues/12

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -116,6 +116,33 @@ For more imformation about the core tests, including what tests are available, h
...
+* [**WooCommerce API**](https://www.npmjs.com/package/woocommerce-api)
+* [**WooCommerce Core E2E Tests**](https://www.npmjs.com/package/@woocommerce/e2e-core-tests)
+
+Examples of how they've integrated these packages and built out tests can be found [in the specs folder](https://github.com/prasidhda/woo-manage-fraud-orders/tree/master/tests/e2e/specs) within the project.
+
+**WooCommerce Payments**
+
+WooCommerce Payments is using both the [**WooCommerce E2E Utils**](https://www.npmjs.com/package/@woocommerce/e2e-utils) and [**WooCommerce API**](https://www.npmjs.com/package/woocommerce-api) packages as part of the [WooCommerce Payments end-to-end test suite](https://github.com/Automattic/woocommerce-payments/tree/develop/tests/e2e). An example usage can be found in the [`Order > Refund Failure` test suite](https://github.com/Automattic/woocommerce-payments/blob/develop/tests/e2e/specs/merchant/merchant-orders-refund-failures.spec.js).
+
+**WooCommerce Bookings**
```
Comment (rrennick) [inline README.md]: @zhongruige This is a private repo and not usable as a reference.

File: README.md [hunk 2]
```diff
@@ -116,6 +116,33 @@ For more imformation about the core tests, including what tests are available, h
...
+
+This section provides examples of plugins and packages that are implementing WooCommerce related end-to-end tests.
+
+**Woo Manage Fraud Orders**
+
+[Woo Manage Fraud Orders](https://github.com/prasidhda/woo-manage-fraud-orders) is making use of the following packages in their end-to-end tests:
+
+* [**WooCommerce E2E Environment**](https://www.npmjs.com/package/@woocommerce/e2e-environment)
+* [**WooCommerce E2E Utils**](https://www.npmjs.com/package/@woocommerce/e2e-utils)
+* [**WooCommerce API**](https://www.npmjs.com/package/woocommerce-api)
```
Comment (rrennick) [inline README.md]:
```suggestion
* [**WooCommerce API**](https://www.npmjs.com/package/@woocommerce/api)
```

File: README.md [hunk 3]
```diff
@@ -116,6 +116,33 @@ For more imformation about the core tests, including what tests are available, h
...
+* [**WooCommerce Core E2E Tests**](https://www.npmjs.com/package/@woocommerce/e2e-core-tests)
+
+Examples of how they've integrated these packages and built out tests can be found [in the specs folder](https://github.com/prasidhda/woo-manage-fraud-orders/tree/master/tests/e2e/specs) within the project.
+
+**WooCommerce Payments**
+
+WooCommerce Payments is using both the [**WooCommerce E2E Utils**](https://www.npmjs.com/package/@woocommerce/e2e-utils) and [**WooCommerce API**](https://www.npmjs.com/package/woocommerce-api) packages as part of the [WooCommerce Payments end-to-end test suite](https://github.com/Automattic/woocommerce-payments/tree/develop/tests/e2e). An example usage can be found in the [`Order > Refund Failure` test suite](https://github.com/Automattic/woocommerce-payments/blob/develop/tests/e2e/specs/merchant/merchant-orders-refund-failures.spec.js).
```
Comment (rrennick) [inline README.md]: Same URL error as above comment.

Also, The formatting for listing packages should be the same throughout the section.

**Review by rrennick — CHANGES_REQUESTED** [review]
@zhongruige This is a nice addition to the documentation. It should be good for merge with the changes I've suggested.

File: README.md (hunk 2 above)
Comment (zhongruige) [reply README.md]: Good call out, I fixed all instances of this in the README.

File: README.md (hunk 1 above)
Comment (zhongruige) [reply README.md]: Removed.

File: README.md (hunk 3 above)
Comment (zhongruige) [reply README.md]: Done, I adjusted the list of packages used to be in a bulleted-list and re-worked the paragraph a bit to accommodate the changes. Let me know what you think.

**Review by rrennick — APPROVED** [review]
Looks good.

**Pull request merged**